sqlserver/datadog_checks/sqlserver/const.py

```diff
@@ -268,3 +268,73 @@
 ]
 
 PROC_CHAR_LIMIT = 500
+
+# Schemas
+DEFAULT_SCHEMAS_COLLECTION_INTERVAL = 1200
+
+DB_QUERY = """
+SELECT
+    db.database_id AS id, db.name AS name, db.collation_name AS collation, dp.name AS owner
+FROM
+    sys.databases db LEFT JOIN sys.database_principals dp ON db.owner_sid = dp.sid
+WHERE db.name = '{}';
+"""
+
+SCHEMA_QUERY = """
+SELECT
+    s.name AS name, s.schema_id AS id, dp.name AS owner_name
+FROM
+    sys.schemas AS s JOIN sys.database_principals dp ON s.principal_id = dp.principal_id
+WHERE s.name NOT IN ('sys', 'information_schema')
+"""
+
+TABLES_IN_SCHEMA_QUERY = """
+SELECT
+    object_id AS id, name
+FROM
+    sys.tables
+WHERE schema_id={}
```

Comment:
```diff
- WHERE schema_id={}
+ WHERE schema_id=?
```


When it comes to _values_, we should use the client's parameter replacement with `?` instead of using python string `format`. Sometimes we have to use `format` because we are generating SQL identifiers. But that should be the exception, not the standard.

Reply:
I see , I will modify for this one. All others will be using IN though.